widget/playground/src/containers/FunctionalLayout/FunctionalLayout.Liquidities.tsx

```diff
@@ -34,6 +34,7 @@ export function LiquiditiesSection() {
 
   const liquiditiesList = (type: LiquidityType) =>
     uniqueSwappersGroup
+      .filter((swapper) => swapper.enabled)
```

Comment:
Is this really necessary? You've already filtered out the disabled swappers in the data slice, which serves as our source of truth.

Reply:
I filtered disabled swappers in the data section in the widget, while this is in the playground. However, I made a change and now filter disabled swappers directly in the data section of the playground.